src/index.html

```diff
@@ -11,6 +11,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="/" 
```

Comment:
better to use # here or the link from the previous task
when you submit this form, error 404 appears

Reply:
Okay! Thank you!